### PR TITLE
Fix SVACE issues

### PR DESCRIPTION
**Pull request by sung-su**
### Description of Change ###
Protection code for nullable values.

### CODE REVIEW

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs [hunk 1]
```diff
@@ -37,7 +38,11 @@ protected override void OnElementChanged(ElementChangedEventArgs<CircleImage> e)
         {
             if (Control == null)
             {
-                _nativeParent = Platform.GetRenderer(Element.Parent).NativeView;
+                if (Element.Parent != null)
+                    _nativeParent = Platform.GetRenderer(Element.Parent).NativeView;
+                else
+                    _nativeParent = XForms.NativeParent;
+                _circleImage = new ElmSharp.EvasImage(_nativeParent);
```
Comment (jkpu) [inline src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs:45]: code duplicated.

                _circleImage = new ElmSharp.EvasImage(_nativeParent);
                _circleImage = new ElmSharp.EvasImage(_nativeParent);

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs [hunk 2]
```diff
@@ -38,6 +39,9 @@ protected override void OnElementChanged(ElementChangedEventArgs<CircleImage> e)
             if (Control == null)
             {
                 _nativeParent = Platform.GetRenderer(Element.Parent).NativeView;
+                if (_nativeParent == null)
+                    _nativeParent = XForms.NativeParent;
```
Comment (myroot) [inline src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs:43]: Why null check? before submit it?

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs (hunk 1 above)
Comment (rookiejava) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs:44]: Please remove this line.

Comment (sung-su) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs:44]: removed

**Comment by sung-su** [comment]
Change reference when creating a CircleImageRenderer

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs [hunk 3]
```diff
@@ -24,25 +24,24 @@
...
         ElmSharp.EvasImage _circleImage;
-        ElmSharp.EvasObject _nativeParent;
 
         protected override void OnElementChanged(ElementChangedEventArgs<CircleImage> e)
         {
             if (Control == null)
             {
-                _nativeParent = Platform.GetRenderer(Element.Parent).NativeView;
-                _circleImage = new ElmSharp.EvasImage(_nativeParent);
+                _circleImage = new ElmSharp.EvasImage(XForms.BaseLayout);
```
Comment (myroot) [inline src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs:40]:
```suggestion
                _circleImage = new ElmSharp.EvasImage(XForms.NativeParent);
```

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs [hunk 4]
```diff
@@ -24,25 +24,24 @@
...
                 _circleImage.IsFilled = true;
                 GetClipImage();
 
-                var image = new FormsNative.Image(_nativeParent);
+                var image = new FormsNative.Image(XForms.BaseLayout);
```
Comment (myroot) [inline src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs:44]:
```suggestion
                var image = new FormsNative.Image(XForms.NativeParent);
```

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs (hunk 3 above)
Comment (sung-su) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs:40]: updated

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs (hunk 4 above)
Comment (sung-su) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleImageRenderer.cs:44]: updated

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/InformationPopupImplementation.cs [hunk 5]
```diff
@@ -255,7 +255,7 @@ void UpdateButton()
 
                 _bottomButton.Clicked += (s, e) =>
                 {
-                    ((IMenuItemController)BottomButton).Activate();
+                    ((IMenuItemController)BottomButton)?.Activate();
```
[the remark this answers is not in the document]
Comment (sung-su) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/InformationPopupImplementation.cs:258]: updated

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/TwoButtonPopupImplementation.cs [hunk 6]
```diff
@@ -259,7 +259,7 @@ void UpdateFirstButton()
 
                 _firstButton.Clicked += (s, e) =>
                 {
-                    ((IMenuItemController)FirstButton).Activate();
+                    ((IMenuItemController)FirstButton)?.Activate();
```
[the remark this answers is not in the document]
Comment (sung-su) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/TwoButtonPopupImplementation.cs:262]: updated

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/TwoButtonPopupImplementation.cs [hunk 7]
```diff
@@ -305,7 +305,7 @@ void UpdateSecondButton()
 
                 _secondButton.Clicked += (s, e) =>
                 {
-                    ((IMenuItemController)SecondButton).Activate();
+                    ((IMenuItemController)SecondButton)?.Activate();
```
[the remark this answers is not in the document]
Comment (sung-su) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/TwoButtonPopupImplementation.cs:308]: updated

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/InformationPopupImplementation.cs [hunk 8]
```diff
@@ -255,7 +255,7 @@ void UpdateButton()
 
                 _bottomButton.Clicked += (s, e) =>
                 {
-                    ((IMenuItemController)BottomButton).Activate();
+                    (IMenuItemController as BottomButton)?.Activate();
```
Comment (myroot) [inline src/Tizen.Wearable.CircularUI.Forms.Renderer/InformationPopupImplementation.cs:258]:
```suggestion
                    (BottomButton as IMenuItemController)?.Activate();
```

Comment (sung-su) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/InformationPopupImplementation.cs]: sorry ;(

**Pull request closed without merging**